python/benchmark/types/iou_benchmark.py

```diff
@@ -0,0 +1,61 @@
+import datetime
```

Comment:
Good work! Could you add Apache license ?

Also run `isort` to put the imports into different sections?